zeus/optimizer/pipeline_frequency/common.py

```diff
@@ -128,14 +143,17 @@ class RankInfo(BaseModel):
         dp_rank: Data parallel rank of the reporting procees.
         pp_rank: Pipeline parallel rank of the reporting procees.
         tp_rank: Tensor parallel rank of the reporting procees.
-        available_frequencies: List of available frequencies for the rank's GPU.
+        available_frequencies: list of available frequencies for the rank's GPU.
+        pipe_schedule: Pipeline schedule (list of strings) for the rank.
+            For example, ["forward", "backward"].
     """
 
     rank: int = Field(ge=0)
     dp_rank: int = Field(ge=0)
     pp_rank: int = Field(ge=0)
     tp_rank: int = Field(ge=0)
     available_frequencies: list[int]
+    pipe_schedule: list[str] = []
```

Comment:
Question: Is this ever okay to be empty (the default `[]`)? Like for the profiler?